src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -752,6 +752,24 @@ public static TestDevice GetTestDevice(this IApp app)
 			return aaa.Config.GetProperty<TestDevice>("TestDevice");
 		}
 
+		/// <summary>
+		/// Sets light device's theme
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		public static void SetLightTheme(this IApp app)
+		{
+			app.CommandExecutor.Execute("setLightTheme", ImmutableDictionary<string, object>.Empty);
+		}
+
+		/// <summary>
+		/// Sets dark device's theme
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		public static void SetDarkTheme(this IApp app)
```

Comment:
Could we include a small test using these added new options?. For example, verify a snapshot with light theme, change it and validate again.

Reply:
Done :)